Amazon.QLDB.Driver.Tests/AsyncTransactionExecutorTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace Amazon.QLDB.Driver.Tests
+{
+    using System.Threading.Tasks;
+    using System.Collections.Generic;
+    using Amazon.IonDotnet.Tree;
+    using Amazon.IonDotnet.Tree.Impl;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Moq;
+
+    [TestClass]
+    public class AsyncTransactionExecutorTests
+    {
+        private static AsyncTransactionExecutor asyncTransactionExecutor;
+        private static readonly string query = "my query";
+        private static Mock<IAsyncResult> mockResult;
+        private static Mock<AsyncTransaction> mockTransaction;
+
+        [ClassInitialize]
+#pragma warning disable IDE0060 // Remove unused parameter
+        public static void Setup(TestContext context)
+#pragma warning restore IDE0060 // Remove unused parameter
+        {
+            mockResult = new Mock<IAsyncResult>();
+            mockTransaction = new Mock<AsyncTransaction>(It.IsAny<Session>(), It.IsAny<string>(), null, null);
+            mockTransaction.Setup(t => t.Abort()).Returns(Task.CompletedTask);
+            mockTransaction.Setup(t => t.Execute(It.IsAny<string>())).ReturnsAsync(mockResult.Object);
+            mockTransaction.Setup(transaction => transaction.Execute(It.IsAny<string>(), It.IsAny<List<IIonValue>>()))
```

Comment:
In Lambdas, let's use the shorter `txn` rather than the longer `transaction`.

Reply:
changed them to `t`